### PR TITLE
doc: add FAQ question about pretty printing

### PR DESCRIPTION
**Pull request by LakshyAAAgrawal**

### CODE REVIEW

**Comment by monperrus** [comment]
Thanks a lot @LakshyAAAgrawal , Improved it. Will merge.

File: doc/faq.md [hunk 1]
```diff
@@ -78,3 +78,6 @@ final String[] builder = new JDTBuilderImpl() //
 		.sources(new SourceOptions().sources(".")) //
 		.build();
 ```
+
+#### What does `Launcher#prettyprint()` do?
+```launcher.prettyprint``` prints the model on disk. The model could have been transformed or not, and is saved in an output directory which one can set up with ```launcher.setSourceOutputDirectory()```. If you want the pretty-print to the standard output (console STDOUT), use method `toString` instead, defined for every CtElement. Reference: https://github.com/INRIA/spoon/issues/4274
```
Comment (algomaster99) [inline doc/faq.md:83]:
```suggestion
```launcher.prettyprint``` prints the model on disk. The model could have been transformed or not, and is saved in an output directory which one can set up with ```launcher.setSourceOutputDirectory()```. The default output directory is called `spooned`. If you want the pretty-print to the standard output (console STDOUT), use method `toString` instead, defined for every CtElement. Reference: https://github.com/INRIA/spoon/issues/4274
```

**Review by algomaster99 — CHANGES_REQUESTED** [review]
Thanks for your PR. I think it is worthwhile to mention the default output directory as well.

**Comment by monperrus** [comment]
Thanks a lot @LakshyAAAgrawal  for the doc PR! Thanks @algomaster99 for your attention to details.

**Pull request merged**